src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs

```diff
@@ -14,26 +14,35 @@
 
 namespace Microsoft.PowerShell.EditorServices.Services.PowerShellContext
 {
+    using System.IO;
     using System.Management.Automation;
 
     internal class PSReadLinePromptContext : IPromptContext
     {
-        private const string ReadLineInitScript = @"
+        private static readonly string _psReadLineModulePath = Path.Combine(
+            Path.GetDirectoryName(typeof(PSReadLinePromptContext).Assembly.Location),
+            "..",
+            "..",
+            "..",
+            "PSReadLine");
+
+        private static readonly string ReadLineInitScript = $@"
             [System.Diagnostics.DebuggerHidden()]
             [System.Diagnostics.DebuggerStepThrough()]
             param()
-            end {
+            end {{
                 $module = Get-Module -ListAvailable PSReadLine |
-                    Where-Object { $_.Version -gt '2.0.0' -or ($_.Version -eq '2.0.0' -and -not $_.PrivateData.PSData.Prerelease) } |
+                    Where-Object {{ $_.Version -ge '2.0.2' }} |
                     Sort-Object -Descending Version |
                     Select-Object -First 1
-                if (-not $module) {
-                    return
-                }
+                if (-not $module) {{
+                    Import-Module {_psReadLineModulePath}
```

Comment:
Does this path need escaping etc to make sure spaces and quotes and weird characters work with it?

Reply:
fixed